Documentation/ConfigFile.md

```diff
@@ -70,6 +70,8 @@ Each key corresponding to a SwiftGen subcommands (`colors`, `coredata`, `fonts`,
 | `output` | Path | **Deprecated** The path of the output file to generate. _(Note: Any intermediate directory up to this file must already exist.)_ |
 | `params` | Dictionary | **Deprecated** Any optional parameter you want to pass to the template (similarly to `--param` in the CLI). |
 
+> 💡 Note: For custom filters, use `.*` to match any characters, and don't forget to escape the dot (`\.`) if you want to match a literal dot like for an extension. Add `$` at the end to ensure the path ends with the extension you want. For example, use `.*\.xib$` to match files with a `.xib` extension.
```

Comment:
I've changed it to `.+`, as it better represents what we're trying to achieve here (match a file extension, with a non-empty filename). We always (?) need a non-empty filename, and this is "general" enough to cover 99% of the cases.

If someone really wants to have a `A/B/C/.json`, they can try (probably won't work, depends on the parser).

Reply:
But again, `.+\.json$` will only be guaranteed to match non-empty filenames if we match on basenames :wink: Again, if we match on the whole path, `.+` will match `A/B/C/` in your example above and `A/B/C/.json` will match the filter anyway… So that `.+` would be enough either. You probably want `[^/]\.json$` instead, so a `.json` at the end of a string preceded by one character there is not a `/` (there might be more characters before, but we don't really care, as long as we match the last character before the extension, we know it's a non-empty filename, so no need to make the RegEx engine backtrack with a `+` or `*` for nothing)